src/kv/Makefile.am

```diff
@@ -26,14 +26,12 @@ NPROC = nproc
 endif
 
 # explicitly disable jemalloc unless we are using it.
-if WITH_JEMALLOC
-DISABLE_JEMALLOC = 0
-else
-DISABLE_JEMALLOC = 1
+if !WITH_JEMALLOC
+export DISABLE_JEMALLOC=1
 endif
 
 rocksdb/librocksdb.a:
-	cd rocksdb && CC="${CC}" CXX="${CXX}" EXTRA_CXXFLAGS="${ROCKSDBCXX_FLAGS}" PORTABLE=1 DISABLE_JEMALLOC="${DISABLE_JEMALLOC}" ${MAKE} -j$(shell ${NPROC}) static_lib
+	cd rocksdb && CC="${CC}" CXX="${CXX}" EXTRA_CXXFLAGS="${ROCKSDBCXX_FLAGS}" PORTABLE=1 ${MAKE} -j$(shell ${NPROC}) static_lib
```

Comment:
@cbodley if we put something like
```
a="FOO=bar"
all:
  ${a} make
```
Makefile tries to add the quotes around "FOO=bar". and it will call "FOO=bar" instead, and "make" will be "argv[1]".

but seems cmake does not suffer from this.